internal/runners/use/show.go

```diff
@@ -0,0 +1,40 @@
+package use
+
+import (
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+type Show struct {
+	out output.Outputer
+	cfg *config.Instance
+}
+
+func NewShow(prime primeable) *Show {
+	return &Show{
+		prime.Output(),
+		prime.Config(),
+	}
+}
+
+func (s *Show) Run() error {
+	projectDir := s.cfg.GetString(constants.GlobalDefaultPrefname)
+	if projectDir == "" {
+		return locale.NewInputError("err_use_show_no_default_project", "No default project is set.")
+	}
+
+	proj, err := project.FromPath(projectDir)
+	if err != nil {
+		return locale.WrapError(err, "err_use_show_get_project", "Could not get default project.")
+	}
+
+	s.out.Print(locale.Tl("use_show", "The default project to use is {{.V0}}, located at {{.V1}}",
+		proj.NamespaceString(),
+		projectDir,
+	))
```

Comment:
We'll want this to be JSON interpretable also. Try something like this: 

https://github.com/ActiveState/cli/blob/37e1f4f6b6dbffed4ba4716f365c8967b8b4834d/internal/runners/export/jwt.go#L32